app/javascript/gobierto_visualizations/modules/contracts_controller.js

```diff
@@ -295,7 +302,7 @@ export class ContractsController {
       _contractsData,
       ({ final_amount_no_taxes = 0 }) => parseFloat(final_amount_no_taxes)
     );
-    const largerContractAmountPct = largerContractAmount / sumContracts;
+    const largerContractAmountPct = (largerContractAmount / sumContracts) || 0;
```

Comment:
Possible division by zero

```suggestion
    const largerContractAmountPct = sumContracts ? (largerContractAmount / sumContracts) : 0;
```

Reply:
Done!